test/integration/server_stats.h

```diff
@@ -14,28 +14,36 @@ class IntegrationTestServerStats {
    * @param name counter name.
    * @param value target value.
    */
-  virtual void waitForCounterEq(const std::string& name, uint64_t value) PURE;
+  virtual void
+  waitForCounterEq(const std::string& name, uint64_t value,
```

Comment:
Please add comments documenting the param (and that no timeout is enabled for default zero)